src/ScottPlot5/ScottPlot/Control/EventArgs/BaseEventArgs.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ScottPlot.Control.EventArgs
+{
+    public class BaseEventArgs : System.EventArgs
```

Comment:
I don't think it precludes us from using `Base` as a suffix since that class isn't really intended for use in public APIs, at least not directly. Users aren't going to make a new event args, they may interact with (for example) `MouseDownEventArgs` if we open up the backend's event handlers at some point.

Reply:
This question will come up again: SignalPlotBase or BaseSignalPlot? We have some time to decide